submissions/mustbefail/oop/index.js

```diff
@@ -0,0 +1,88 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(species, name, gender, hands, legs, friends, phrase) {
+    this.friends = friends;
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.hands = hands;
+    this.legs = legs;
+    this.phrase = phrase;
+  }
+  saying() {
+    return this.phrase;
+  }
+}
+
+class Cat extends Inhabitant {
+  constructor(name, gender, friends, phrase) {
```

Comment:
`phrase` is something that every inhabitant has. They may not have friends.
This how the task description puts it.
Please make friends an optional property.